Snippets/Sqs/Sqs_All/QueueDeletion/QueueDeletionUtils.cs

```diff
@@ -0,0 +1,99 @@
+﻿namespace SqsAll.QueueDeletion
+{
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.Runtime;
+    using Amazon.SQS;
+    using Amazon.SQS.Model;
+    using SqsAll.QueueCreation;
+    using Sqs_All;
+
+    #region sqs-delete-queues
+
+    public static class QueueDeletionUtils
+    {
+        public static async Task DeleteAllQueues()
+        {
+            using (var client = ClientFactory.CreateSqsClient())
+            {
+                await DeleteQueues(client)
+                    .ConfigureAwait(false);
+            }
+        }
+
+        static async Task DeleteQueues(IAmazonSQS client)
+        {
+            var queuesToDelete = await client.ListQueuesAsync("")
```

Comment:
I don't think it cross cuts everything. Most other scripts just accept the queue name or the endpoint name. Do you agree that it is enough to just add a prefix param to the deletion scripts?